cmd/percona-dbaas/pxc/create.go

```diff
@@ -57,8 +57,23 @@ var createCmd = &cobra.Command{
 			fmt.Fprintf(os.Stderr, "[ERROR] %v\n", err)
 			return
 		}
+		if len(*labels) > 0 {
+			match, err := regexp.MatchString("([a-zA-Z0-9_]+=[a-zA-Z0-9_]+)|([a-zA-Z0-9_]+=[a-zA-Z0-9_]+,)+([a-zA-Z0-9_]+=[a-zA-Z0-9_]+)", *labels)
```

Comment:
1) only properly formated key-values allowed
2) it can be more than two labels
```suggestion
			match, err := regexp.MatchString("^(([a-zA-Z0-9_]+=[a-zA-Z0-9_]+)(,|$))+$", *labels)
```

Reply:
done